app/views/speakers/actions/_admin.html.erb

```diff
@@ -13,4 +13,14 @@
       <span>Fetch from GitHub</span>
     <% end %>
   <% end %>
+
+  <%= ui_button url: "/admin/resources/speakers/#{@speaker.slug}", target: :_blank, kind: :ghost, size: :sm do %>
```

Comment:
I think you could use the avo path helpers here
```suggestion
  <%= ui_button url: avo.resources_speaker_path(@speaker), target: :_blank, kind: :ghost, size: :sm do %>
```

Reply:
Ah nice, didn't know about this 🙌🏼